client/types/balance_transactions.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface BalanceTransaction {
```

Comment:
Would it make sense to include `BalanceTransaction` in a file called `client/types/transactions.d.ts`? That way, we'll open up the possibility of adding more transaction-related types to this file.

Reply:
Balance Transactions is a separate entity and can be part of many other entities. I decided to have a separate file for it to avoid possibility of circular dependencies hell.